### PR TITLE
Add better logging for info log level

### PR DESCRIPTION
**Pull request by jpbruinsslot**

### CODE REVIEW

**Review by ammar92 — APPROVED** [review]
Nice stuff!

Maybe for discussion later: I would suggest to use more generic names/ terms rather than variable names while logging (e.g. `..p_item.hash=%s..` to `item_hash=%s` or `queue.pq_id=%s` to `queue_id=%s`), because variable names tend to change (and if we are planning to implement structured logging in the near future, you want to change the logging format as little as possibly) and second, too many implementation details leak out.

File: logging.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "disable_existing_loggers": false,
     "formatters": {
         "default": {
-            "format": "%(asctime)s [%(process)d] [%(levelname)s] [%(module)s] [%(name)s] %(message)s",
```
Comment (sigio) [inline logging.json:6]: asctime mag wat mij betreft ook weg... .logging gaat via systemd-journal, en die voegt ook al zijn eigen timestamps toe:
Jan 27 11:02:50 scheduler.acc.kat.prolocation.net scheduler-scheduler-app[1141203]: [2023-01-27 11:02:50 +0100] [1141203] [WARNING] [scheduler] [scheduler.schedulers.scheduler] Queue normalizer-tst is empty, not populating new tasks [queue_id=normalizer-tst, qsize=0]

**Comment by sigio** [comment]
"Queue XXX is empty, not populating new tasks" ook gaarne naar debug of info, staat in de huidige release op warning

**Review by underdarknl — APPROVED** [review]

File: logging.json (hunk 1 above)
Comment (underdarknl) [reply logging.json:6]: This can be handled with a custom json file if this is the case.

**Pull request merged**